c.html.markdown

```diff
@@ -630,6 +630,54 @@ typedef void (*my_fnp_type)(char *);
 
 ```
 
+Header files are an important part of c as they allow for the connection of c 
+source files and can simplify code and definitions by seperating them into 
+seperate files.
+
+Header files are syntaxtically similar to c source files but reside in ".h" 
+files. They can be included in your c source file by using the precompiler 
+command #include "example.h", given that example.h exists in the same directory 
+as the c file.
+
+```c
```

Comment:
That wont work actually. It must be 'c' as that is the syntax highlighting. 

Go ahead and remove the ``` from line 631 and this line here we are talking about and we will get it merged in.